pydash/pydash_app/dashboard/dashboard.py

```diff
@@ -51,6 +55,14 @@ def remove_endpoint(self, endpoint):
         except ValueError:
             raise
 
+    def add_endpoint_call(self, endpoint_call):
+        """
+        Adds an endpoint call to the dashboard.
+        :param endpoint_call: The endpoint call to add
+        """
+        self._endpoint_calls.append(endpoint_call)
+        self._aggregator.add_endpoint_call(endpoint_call)
+
```

Comment:
We should also add the endpoint call to the endpoint in self.endpoints.

Reply:
Good point! Although that would require searching the entire list of endpoints which might take very long. Any ideas to avoid that?

Reply:
we could make the endpoint collection some form of dict. I'm not sure how fast the creation would be, but lookup should be pretty fast.

Reply:
Fixed in new commit.